### ISSUE TITLE
question about learning rate and batchsize

### ISSUE THREAD

**Issue by Aiz-wallenstein** [zylo117/Yet-Another-EfficientDet-Pytorch#555]
thanks for your repo! 
And can you tell me your paramter set : Initial learning rate，batchsize，when you train Effdet-d0. And how many epoch do you train and get  33.1 mAP.  Thanks!
